pkg/proc/threads.go

```diff
@@ -71,6 +71,24 @@ func topframe(g *G, thread Thread) (Stackframe, Stackframe, error) {
 	var frames []Stackframe
 	var err error
 
+	var pc uint64
```

Comment:
What's the problem here with just letting ThreadStacktrace/(*G).Stacktrace get called? I think it should at least be able to return a single frame.

Reply:
Done. Might still panic due to instances of `lineInfo` being used still in the stacktrace code but I'm happy to fix those as they come up in subsequent PRs. I want to let this marinate for a bit anyways prior to cutting a release.